azurerm/internal/services/datafactory/data_factory_linked_service_sftp_resource.go

```diff
@@ -264,6 +283,14 @@ func resourceDataFactoryLinkedServiceSFTPRead(d *schema.ResourceData, meta inter
 		}
 	}
 
+	if skipHostKeyValidation := sftp.SkipHostKeyValidation; skipHostKeyValidation != nil {
```

Comment:
Can we nil check on the `sftp.SftpserverLinkedServiceTypeProperties`?

Reply:
Isn't this checked in line 257?

Reply:
No, tbh. Line 257 assert the polymorphism is a sftp server linked service variant type, while I commented above is to nil check on the struct field `SftpserverLinkedServiceTypeProperties`.